async-nats/src/jetstream/context.rs

```diff
@@ -842,3 +871,58 @@ impl IntoFuture for PublishAckFuture {
         ))
     }
 }
+
+/// Used to for building customized `publish` message.
+#[derive(Default, Clone, Debug)]
+pub struct PublishBuilder {
```

Comment:
Want to infer the suffix?

```suggestion
pub struct Publish {
```

Reply:
I didn't like `Publish`, but after switching `new` to `build` I'm fine with it. ok.